build.gradle

```diff
@@ -11,8 +11,8 @@ plugins {
 
 ext {
     group = 'io.orkes.queue'
-    appVersion = '0.0.2-SNAPSHOT'
-    springBootVersion = '2.5.6'
+    appVersion = '0.0.5-SNAPSHOT'
+    springBootVersion = '3.1.5'
```

Comment:
Note to reviewers - I don't know how this is going to look on release time.
We are in version 1.0.6 https://mvnrepository.com/artifact/io.orkes.queues/orkes-redis-queues - I don't know if this number will impact that versioning.